mypy/typeanal.py

```diff
@@ -223,6 +224,13 @@ def visit_tuple_type(self, t: TupleType) -> Type:
         fallback = t.fallback if t.fallback else self.builtin_type('builtins.tuple', [AnyType()])
         return TupleType(self.anal_array(t.items), fallback, t.line)
 
+    def visit_typeddict_type(self, t: TypedDictType) -> Type:
```

Comment:
This is not sufficient, I think. It seems that we hit the `visit_unbound_type` case instead (see above for a discussion of that), and it should be handled similarly to named tuples.